helical/models/classification/neural_network.py

```diff
@@ -5,7 +5,7 @@
 import tensorflow as tf
 from tensorflow.keras.models import Sequential
 from tensorflow.keras.layers import Dense, Dropout
-from tensorflow.keras.optimizers import Adam
+from tensorflow.keras.optimizers.legacy import Adam
```

Comment:
With using GPU, I got an error in the execution of evaluating cell type annotation. It was recommended to use legacy.Adam. This fixed the issue for me.